jetbrains-core/tst/software/aws/toolkits/jetbrains/core/region/MockRegionProvider.kt

```diff
@@ -57,15 +56,8 @@ class MockRegionProvider : ToolkitRegionProvider() {
         fun getInstance(): MockRegionProvider = ServiceManager.getService(ToolkitRegionProvider::class.java) as MockRegionProvider
     }
 
-    class RegionProviderRule(projectRule: ProjectRule) : ExternalResource() {
-        val regionProvider by lazy {
-            projectRule.project // For ServiceManager to be wired
-            getInstance()
-        }
-
-        override fun before() {
-            regionProvider.reset()
-        }
+    class RegionProviderRule : ExternalResource() {
```

Comment:
Hmm, this should probably extend ApplicationRule() since it needs ServiceManager to be started to be safe